nengo_gui/static/components/netgraph.js

```diff
@@ -307,9 +307,6 @@ Nengo.NetGraph.prototype.on_message = function(event) {
         var item = this.svg_objects[data.uid];
         item.set_label(data.name);
 
-        var item_mini = this.minimap_objects[data.uid];
-        item_mini.set_label(data.name);
-
```

Comment:
Hmm..  I'm still not sure I'm following the logic here right.  @studywolf , could you quickly confirm that it's okay to remove this setting of the label?